### ISSUE TITLE
Fail to paste one folder from a public connected blob container to another blob container

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#3394]
**Storage Explorer Version:** 1.15.0-dev
**Build**: 20200813.10
**Branch**: master
**Platform/OS:** Windows 10/ Linux Ubuntu 18.04/ macOS Catalina
**Architecture:** ia32/x64
**Regression From:** Not a regression

**Steps to reproduce:**

1. Expand one storage account -> Blob Containers.
2. Create one blob container -> Set its public read access to 'Container' -> Upload one folder to it.
3. Attach the blob container via 'Connect to public blob container'.
4. Copy the folder in the public blob container -> Paste it to another non-ADLS Gen2 blob container.
5. Check the result.

**Expect Experience:**
Succeed to paste.

**Actual Experience:**
Fail to paste.
![image](https://user-images.githubusercontent.com/41351993/90233199-5602de80-de50-11ea-8fa9-043d8cd27ef9.png)

Error details:
```
{
  "message": "\"failed to perform copy command due to error: no SAS token or OAuth token is present and the resource is not public\""
}
```

**More  Info:**
1. Fail to download several blobs in one folder under the public connected blob container.
![image](https://user-images.githubusercontent.com/41351993/90234770-ca3e8180-de52-11ea-876d-15a8575f6e3e.png)

2. Show the below errors when trying to paste one file or folder from a public connected blob container to one ADLS Gen2 blob container.
![image](https://user-images.githubusercontent.com/41351993/90233831-4e900500-de51-11ea-9195-4fd0c008764d.png)

**Comment by MRayermannMSFT** [comment]
known issue, AzCopy will fix in future release

**Comment by v-xianya** [comment]
Verified on build 20200916.1. Reproduce.

**Comment by JasonYeMSFT** [comment]
@v-xianya We have integrated a new version of AzCopy in 1.16.0, could you please help verify this issue to see if it is fixed?

**Comment by v-xianya** [comment]
@JasonYeMSFT This issue is fixed on build main/20201112.8.
For the below behaviors, is that expected?

Show "Failed to start transfer: Insufficient credentials." when trying to paste one file or folder from a public connected blob container to one ADLS Gen2 blob container.
![image](https://user-images.githubusercontent.com/41351993/99020658-0a201c80-259a-11eb-98ba-4cb13065d318.png)

**Comment by v-limu** [comment]
Verified on build 20201121.3. Succed to paste one file or folder from a public connected blob container to one ADLS Gen2 blob container. So close it.
